_posts/en/newsletters/2021-10-06-newsletter.md

```diff
@@ -0,0 +1,167 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding (nothing at this time)
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIPs)][bips repo], and
+[Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core GUI #416][] adds an "Enable RPC server" checkbox which allows
+  the user to turn on and off Bitcoin Core's RPC server (requires restart).
+
+    {:.center}
+    ![Screenshot of the Enable RPC server configuration option](/img/posts/2021-10-gui-rpc-server.png)
+
+- [Bitcoin Core #20591][] changes the wallet time calculation logic to
+  only use the block timestamp when rescanning historical blocks for
+transactions relevant to the wallet. Users and applications using the
+`rescanblockchain` RPC to manually invoke rescans should no longer see
+transactions inaccurately labeled with the time scanned instead of
+time confirmed, a bug diagnosed and fixed by a patient downstream
+developer looking after the Bitcoin ecosystem.
+
+- [Bitcoin Core #22722][] updates the `estimatesmartfee` RPC so that it
+  only returns feerates higher than both the configured and dynamic
+  minimum transaction relay fees.  For example, if the estimator
+  calculates a fee of 1 sat/vbyte, the configured value is 2 sat/vbyte,
+  and the dynamic minimum has risen to 3 sat/vbyte, then a value of 3
+  sat/vbyte will be returned.
+
+- [Bitcoin Core #17526][] adds the Single Random Draw (SRD) algorithm as
+  a third coin selection strategy. The wallet will now acquire coin
+  selection results from each the Branch and Bound (BnB), the knapsack,
+  and the SRD algorithm and use the [waste heuristic][news165 waste] we
+  described previously to select the most cost effective coin selection
+  result out of the three to fund the transaction.
+
+  In simulations based on about 8,000 payments, the PR author
+  found that the addition of the SRD algorithm reduced overall
+  transaction fees by 6% and increased the occurrence of changeless
+  transactions from 5.4% to 9.0%. Not creating a change output decreases
+  the weight and fees of a transaction, reduces the wallet's UTXO pool
+  size, saves the cost of later spending the change output, and is
+  thought to improve the privacy of the wallet.
+
+- [Bitcoin Core #23061][] fixes the `-persistmempool` configuration
+  option, which previously did not persist the mempool to disk on
+  shutdown when no parameter was passed (to actually persist, you had to
+  pass `-persistmempool=1`).  Now using the bare option will persist the
+  mempool.
```

Comment:
May be worth noting that the default value is to persist the mempool, so users should never have to explicitly pass `-persistmempool`.